src/BlockTypes/ProductCategories.php

```diff
@@ -109,9 +121,34 @@ protected function get_container_classes( $attributes = array() ) {
 			$classes[] = 'is-list';
 		}
 
-		return implode( ' ', $classes );
+		$line_height_class_and_style = StyleAttributesUtils::get_line_height_class_and_style( $attributes );
+		$text_color_class_and_style  = StyleAttributesUtils::get_text_color_class_and_style( $attributes );
+		$font_size_class_and_style   = StyleAttributesUtils::get_font_size_class_and_style( $attributes );
+
+		$line_height_class = isset( $line_height_class_and_style['class'] ) ? $line_height_class_and_style['class'] : null;
+		$text_color_class  = isset( $text_color_class_and_style['class'] ) ? $text_color_class_and_style['class'] : null;
+		$font_size_class   = isset( $font_size_class_and_style['class'] ) ? $font_size_class_and_style['class'] : null;
+
+		$classes = array_filter(
+			[ $line_height_class, $text_color_class, $font_size_class ]
```

Comment:
We should use only one array style. Same for line 141 below.

Reply:
What do you mean with `use only one array style`. Maybe do you mean that I should use `array`?

```suggestion
			array($line_height_class, $text_color_class, $font_size_class)
```

Reply:
@gigitux  Yeah, I mean the syntax, we should use one style to define arrays.